message/validation/common_checks.go

```diff
@@ -98,18 +98,24 @@ func (mv *messageValidator) validateBeaconDuty(
 	slot phase0.Slot,
 	indices []phase0.ValidatorIndex,
 ) error {
+	if len(indices) != 1 {
+		return ErrProposalSCCSeveralIndices
+	}
+
+	index := indices[0]
```

Comment:
from what I've seen in the code, we are getting the indicies from the committeeInfo struct which is created by getting all the committee indices from the validatorStore, I don't userstand how can you be sure that the first index is the correct validator that has the proposal duty?